docs/docusaurus.config.js

```diff
@@ -0,0 +1,143 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require("./src/themes/prism-testkube-light");
+const darkCodeTheme = require("./src/themes/prism-testkube-dark");
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: "Testkube Documentation",
+  tagline:
+    "Your somewhat opinionated and friendly Kubernetes testing framework",
+  url: "https://testkube.kubeshop.io",
+  baseUrl: "/",
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+  favicon: "img/logo.svg",
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: "kubeshop", // Usually your GitHub org/user name.
+  projectName: "testkube", // Usually your repo name.
+
+  // Even if you don't use internalization, you can use this field to set useful
+  // metadata like html lang. For example, if your site is Chinese, you may want
+  // to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"],
+  },
+
+  presets: [
+    [
+      "classic",
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          routeBasePath: "/",
```

Comment:
Same as above

Reply:
nope... this should NOT be changed – otherwise we'll have `/testkube/teskube` links :)